docs/content/guides/accessibility/accessibility.md

```diff
@@ -0,0 +1,185 @@
+| [**enterBeginsEditing**](https://handsontable.com/docs/javascript-data-grid/api/options/#enterbeginsediting) | -    | -          |
+| [**autoWrapCol**](https://handsontable.com/docs/javascript-data-grid/api/options/#autowrapcol)         | -       | -          |
+| [**autoWrapRow**](https://handsontable.com/docs/javascript-data-grid/api/options/#autowraprow)         | -       | -          |
+| [**renderAllRows**](https://handsontable.com/docs/javascript-data-grid/api/options/#renderallrows)     | -       | -          |
+| [**viewportColumnRenderingOffset**](https://handsontable.com/docs/javascript-data-grid/api/options/#viewportcolumnrenderingoffset) | - | -  |
+
+
+
+## High-contrast Theme
+Ensuring that content is easily readable and distinguishable is important for users, especially those with visual impairments. Proper contrast between elements not only enhances readability but also ensures that users can comfortably interact with the content for extended periods. The minimum recommended contrast between images and text, or a background and text, is [4.5:1](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum). To meet this requirement we recommend either:
+
+- Overriding the default Handsontable theme with custom CSS values.
+- Use the special software, like the [High Contrast](https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph) plugin maintained by Google for Chrome users.
+
+
+## Requirements for the Developers
+
+When you customize Handsontable, it's you who is responsible for ensuring the accessibility of your solution. Especially when it comes to creating [custom cell type](https://handsontable.com/docs/javascript-data-grid/cell-function/) or [custom plugin](https://handsontable.com/docs/javascript-data-grid/custom-plugins/), it’s important to remember about making them accessible to everyone, best if they meet **WCAG 2.1 AA guidelines** at minimum.
+
+Recommendations for Customizations:
+
+- Test your code against [WCAG 2.1 requirements](https://www.w3.org/WAI/WCAG21/quickref/).
+- Use proper color contrast, font size, and semantic HTML.
+- If needed, implement additional **WAI-ARIA** attributes.
+- Avoid flashing or blinking content.
+- Test your customizations with real users who have different types of disabilities. If you don’t have access to real users, try [Funkify](https://www.funkify.org/), a disability simulator, which can help you step into disabled users' shoes.
+
+> **Note:** The accessibility level of any component in your application may be decreased by a low accessibility level of its parent elements. For this reason, make sure to always check the accessibility of the entire page, using tools such as [Lighthouse](https://developers.google.com/web/tools/lighthouse).
+
+## Known Limitations
+
+   - **ARIA Labels:** We are actively working towards providing comprehensive translations. We appreciate your patience and will keep our users updated on the progress.
```

Comment:
Its worth linking to our documentation on how to add locales https://handsontable.com/docs/javascript-data-grid/language/